Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -25,62 +25,85 @@ NS_ASSUME_NONNULL_BEGIN
 @interface STPPaymentConfiguration : NSObject<NSCopying>
 
 /**
- This is a convenience singleton configuration that uses the default values
- for every property
+ This is a convenience singleton configuration that uses the default values for
+ every property
  */
 + (instancetype)sharedConfiguration;
 
 /**
- *  Your Stripe publishable key. You can get this from https://dashboard.stripe.com/account/apikeys .
+ Your Stripe publishable key
+ 
+ @see https://dashboard.stripe.com/account/apikeys
  */
-@property(nonatomic, copy)NSString *publishableKey;
+@property (nonatomic, copy, readwrite) NSString *publishableKey;
 
 /**
- *  An enum value representing which payment methods you will accept from your user in addition to credit cards. Unless you have a very specific reason not to, you should leave this at the default, `STPPaymentMethodTypeAll`.
+ An enum value representing which payment methods you will accept from your user
+ in addition to credit cards. Unless you have a very specific reason not to, you
+ should leave this at the default, `STPPaymentMethodTypeAll`.
  */
-@property(nonatomic)STPPaymentMethodType additionalPaymentMethods;
+@property (nonatomic, assign, readwrite) STPPaymentMethodType additionalPaymentMethods;
```

Comment:
mm okay, lets keep going down this explicitness path. I don't think it's ever going to be reasonable for objc code to fit under 80 characters because of the typical verbose naming conventions.